src/components/exercise/mode.cjsx

```diff
@@ -44,8 +44,11 @@ ExMode = React.createClass
     @setState(nextAnswers) unless _.isEmpty(nextAnswers)
 
   focusBox: ->
-    {focus, mode} = @props
-    @refs.freeResponse?.getDOMNode?().focus?() if focus and mode is 'free-response'
+    {focus} = @props
+    if focus
+      @refs.freeResponse?.getDOMNode?().focus?()
+    else
+      @refs.freeResponse?.getDOMNode?().blur?()
 
```

Comment:
since the `focusBox` method is sometimes actually doing the opposite of focus, should we rename it to `setBoxFocusState` or something?  Also I see some logic in `componentDidUpdate` that conditionally calls `@focusBox()`  That logic could perhaps be moved into the method?